packages/fxa-profile-server/lib/routes/avatar/post-from-auth-server.js

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const assert = require('assert');
+
+const Joi = require('@hapi/joi');
```

Comment:
Should this be "joi" (like it is below in packages/fxa-profile-server/lib/routes/avatar/upload-from-auth-server.js#7)